invenio_records_resources/services/records/service.py

```diff
@@ -54,10 +54,7 @@ def indexer(self):
 
     def record_to_index(self, record):
         """Function used to map a record to an index."""
-        # We are returning "_doc" as document type as recommended by
-        # the search engine documentation. In v8
-        # document types will have been completely removed.
-        return record.index._name, "_doc"
+        return record.index._name
```

Comment:
is this a breaking change?

Reply:
no, not really... it wasn't required in ES for a while now (certainly not in ES7, which is the minimum supported version in invenio nowadays), and is actually incompatible with OSv2.
`invenio-indexer` actually still handles [both cases](https://github.com/inveniosoftware/invenio-indexer/commit/0f28228e9f0700307c7d9574f68acd4ff87d1599#diff-3f68cfb18ee1c14da652ccebd99987da87e79b89c87fa6d5bb75665eee508a13R102), either returning only one value or a tuple – however, in the latter case it issues a warning, which this change fixes.

Reply:
It's a legacy from ES6 IIRC. It was made "optional" in ES7. See [here](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/removal-of-types.html)